app/routes/docs.tsx

```diff
@@ -30,7 +30,7 @@ const DirectoryLink = ({
     <Link
       to={d.path.replace(/\/?index$/, "")}
       className={`w-full py-1 px-4 cursor-pointer hover:bg-gray-200 capitalize ${
-        pathSelected === d.path ? "bg-gray-100" : ""
+        pathSelected.replace(/^docs\//, "") === d.path ? "bg-gray-100" : ""
```

Comment:
This won't work for me, because my slashes are backwards, so I can't double check it.

`pathSelected=/docs/basics/terminology` vs `path=basics\terminology`

But I think this will fix it for the live site.

Reply:
```suggestion
        pathSelected.replace(/^docs[\\/]/, "") === d.path ? "bg-gray-100" : ""
```

does making the path slash optional like this work?

Reply:
Ah!  
For the `docs/docs` issue it does.
However, I believe I still won't see the `bg-gray-100` because my `d.path` is with `\`'s

Reply:
Ah can we then normalize `d.path` so that it's always `/`? As in from the backend so that the front end is consistent

Reply:
so it looks like `d.path` is normalized with `readdirSync` earlier on, so I normalized `pathSelected`.
I noticed there's an additional slash at the beginning of `pathSelected`, so added that to the regex.

But then I noticed that the `bg-gray-100` only applied if you loaded that page.  The directory wasn't getting refreshed.  So I tried adding `pathSelected` to state (https://github.com/samepage-network/samepage.network/pull/87/commits/9f4c08d7137653da04130aab9235227f6f462082) but that doesn't remove the `bg-gray-100`, it just adds it to each directory clicked.

![image](https://github.com/samepage-network/samepage.network/assets/3792666/a90a3616-f0a1-4c57-9b07-c5993d1778f1)


I've been at this for awhile now, so figured I'd upload what I had and get your thoughts.